itest/loadtest/utils.go

```diff
@@ -289,3 +289,18 @@ func getBitcoinConn(t *testing.T, cfg *BitcoinConfig) *rpcclient.Client {
 
 	return client
 }
+
+// stringToAssetType converts a string of an asset type to its respective taprpc
+// type enum value.
+func stringToAssetType(t string) taprpc.AssetType {
+	switch t {
+	case "normal":
+		return taprpc.AssetType_NORMAL
```

Comment:
nit: could just have this as the `default` case, since that's what we do in the end anyway.